src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -263,12 +249,19 @@ static boolean isForwardStrandRepresentation(final AlignmentInterval regionWithL
                                                  final boolean involvesReferenceIntervalSwitch) {
 
         if (strandSwitch == StrandSwitch.NO_SWITCH) {
-            return regionWithLowerCoordOnContig.forwardStrand && regionWithHigherCoordOnContig.forwardStrand;
+            return regionWithLowerCoordOnContig.isForwardStrand && regionWithHigherCoordOnContig.isForwardStrand;
         } else {
             return !involvesReferenceIntervalSwitch;
         }
     }
 
+    public static boolean isNotSimpleTranslocation(final ChimericAlignment chimericAlignment) {
```

Comment:
If we just made this method non-static on Chimeric alignment you wouldn't need this overload of it and calls to it would be much simpler.